CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### citus v9.2.8 (November 4, 2021) ###
+
+* Adds a configure flag to enforce security
```

Comment:
this sounds a bit too vague to me, given that this is an old version it might be okay but I think we should try to have more explanatory entries in the changelogs. Any thoughts @JelteF ?

Reply:
>  I think we should try to have more explanatory entries in the changelogs. 

I agree, but given that this entry was already in the changelog for some other versions, I think we should use the same entry for the 9.2.8 as well.